Error while checking build/O2/fullCI for 7dd01da99593ff64fcd59a0c094086d94d8c75fa at 2024-03-20 15:43:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
/sw/SOURCES/O2/12889-slc8_x86-64/0/Detectors/ITSMFT/common/workflow/src/DeadMapBuilderSpec.cxx:338:11: error: 'alert' is not a member of 'fair::Severity'
/sw/SOURCES/O2/12889-slc8_x86-64/0/Detectors/ITSMFT/common/workflow/src/DeadMapBuilderSpec.cxx:338:11: error: 'alert' is not a member of 'fair::Severity'
/sw/SOURCES/O2/12889-slc8_x86-64/0/Detectors/ITSMFT/common/workflow/src/DeadMapBuilderSpec.cxx:338:11: error: 'alert' is not a member of 'fair::Severity'
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/12889/7dd01da99593ff64fcd59a0c094086d94d8c75fa/build_O2_fullCI/pretty.html).

--- a/Detectors/ITSMFT/common/workflow/src/DeadMapBuilderSpec.cxx
+++ b/Detectors/ITSMFT/common/workflow/src/DeadMapBuilderSpec.cxx
@@ -278,7 +278,7 @@ void ITSMFTDeadMapBuilder::PrepareOutputCcdb(EndOfStreamContext* ec, std::string
       ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "TimeDeadMap", 1}, info);
     } else {
       ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBPayload, "TimeDeadMap", 0}, *image.get());
-      ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "TimeDeadMap", 0}, info);a
+      ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "TimeDeadMap", 0}, info);
     }
   }
 

--- a/Detectors/ITSMFT/common/workflow/src/DeadMapBuilderSpec.cxx
+++ b/Detectors/ITSMFT/common/workflow/src/DeadMapBuilderSpec.cxx
@@ -278,7 +278,7 @@ void ITSMFTDeadMapBuilder::PrepareOutputCcdb(EndOfStreamContext* ec, std::string
       ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "TimeDeadMap", 1}, info);
     } else {
       ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBPayload, "TimeDeadMap", 0}, *image.get());
-      ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "TimeDeadMap", 0}, info);
+      ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "TimeDeadMap", 0}, info);a
     }
   }
 
@@ -335,7 +335,7 @@ void ITSMFTDeadMapBuilder::stop()
       LOG(warning) << "endOfStream not processed. Sending output to ccdb from the " << detname << "deadmap builder workflow.";
       PrepareOutputCcdb(nullptr, mCCDBUrl);
     } else {
-      LOG(alert) << "endOfStream not processed. Nothing forwarded as output.";
+      LOG(alarm) << "endOfStream not processed. Nothing forwarded as output.";
     }
     isEnded = true;
   }

